packages/generator-particle-base/src/generators/app/index.ts

```diff
@@ -115,20 +101,20 @@ module.exports = class extends Generator {
   async initializing() {
     await this._promptUser()
 
-    // All composed generators must be imported following this syntax https://yeoman.io/authoring/composability.html
-    if (
-      this.configuration.options.frontendFramework.includes(
-        FrontendFrameworkOptions.REACT
-      )
-    ) {
-      this.composeWith(
-        require.resolve('@phase2/generator-particle-storybook'),
-        {
-          configuration: this.configuration,
-          updatePackageJson: this._updatePackageJson,
-        }
-      )
-    }
+    // // All composed generators must be imported following this syntax https://yeoman.io/authoring/composability.html
+    // if (
+    //   this.configuration.options.frontendFramework.includes(
+    //     FrontendFrameworkOptions.REACT
+    //   )
+    // ) {
+    //   this.composeWith(
+    //     require.resolve('@phase2/generator-particle-storybook'),
+    //     {
+    //       configuration: this.configuration,
+    //       updatePackageJson: this._updatePackageJson,
+    //     }
+    //   )
+    // }
```

Comment:
Is this commented out because it will be used later? Is there a need to comment it out?

Reply:
Honestly I'm not using it right now, but it looked useful to keep as documentation for the rest of the sub-generators to come.